patches/0002-Add-Mariner-support-with-FIPS-dependency-flag.patch

```diff
@@ -33,7 +33,8 @@ index 3d8ddc4235a658..0bc708016e8b10 100644
 +{{ ) elif is_cbl_mariner then ( -}}
 +FROM mcr.microsoft.com/cbl-mariner/base/core:{{ cbl_mariner_version }}
 +
-+RUN tdnf install -y \
++RUN tdnf update -y; \
```

Comment:
Seeing a fresh `;` made me realize that this `RUN` is missing `set -eux; \`. `e` is important so that if `tdnf update -y` fails, the whole `RUN` fails, rather than using the exit code of the last command.

Reply:
I'll fix this in a followup PR, and also work on updating our template/Dockerfiles from upstream--there are some other changes.